AnnService/inc/Core/Common/WorkSpace.h

```diff
@@ -135,54 +135,71 @@ namespace SPTAG
         };
 
         class DistPriorityQueue {
-            std::unique_ptr<float[]> m_data;
             int m_size;
+            std::unique_ptr<float[]> m_data;
+            int m_length;
             int m_count;
+            
         public:
             DistPriorityQueue() {}
 
             void Resize(int size_) {
```

Comment:
may add some defense code when size_ <=0,
may consider optimization when size_ <= m_size.

Reply:
resize is only used in initialization. we can add size_ > 0 check in the next submit.